.github/ISSUE_TEMPLATE/new_sample.md

```diff
@@ -21,9 +21,10 @@ Please supply what [Domain](https://github.com/oneapi-src/oneAPI-samples/wiki/Re
 Include a short Description of what the sample while Demonstrate and why it is important to the ecosystem.
 
 ## Dependencies
-Please list any dependencies, libraries and images, etc and urls to the download locations
+Please list any dependencies, libraries and images, etc and urls to the download locations using the following [form](https://teams.microsoft.com/l/file/576476E2-256A-4AF9-A1A9-9619A3DB9EB2?tenantId=46c98d88-e344-4ed4-8496-4ed7712e255d&fileType=pptx&objectUrl=https%3A%2F%2Fintel.sharepoint.com%2Fsites%2FOneAPICodeSamplesCommittee%2FShared%20Documents%2FGeneral%2FNewSample_Approval_Template_v0.7.pptx&baseUrl=https%3A%2F%2Fintel.sharepoint.com%2Fsites%2FOneAPICodeSamplesCommittee&serviceName=teams&threadId=19:d29358239f3343f2b26f0c13a47a478d@thread.skype&groupId=1bc67a43-c3a0-4246-98e2-21f851672b78)
```

Comment:
This link opens MS Teams, is that the intent?  Is this an appropriate link to include in a public repo?

Reply:
removed